product-service/handlers/postProduct.js

```diff
@@ -0,0 +1,47 @@
+import { createProduct } from "../models/product";
+
+const postProduct = async (e) => {
+  console.log('Lambda function has been invoked with event: ', e);
+  const product = JSON.parse(e.body);
```

Comment:
If we not send anything in body, lambda return 500 "Internal server error". I think problem in this line, body is null

Reply:
"All lambdas return error 500 status code on any error (DB connection, any unhandled error in code)" - from task description.
Empty request is unhandled error in code and it returns 500.

Reply:
I think all cases must be handled in code and return meaningful description, but in this situation we receive "Internal server error", that not help us in develop)

Reply:
sure, but this is test project where we are learning aws, not error handling ) part of evaluation was to add status codes on errors, nothing about different descriptions was mentioned:
"All lambdas return error 500 status code on any error (DB connection, any unhandled error in code)"